extensions/blocks/podcast-player/style.scss

```diff
@@ -126,9 +132,15 @@ $player-float-background: $light-gray-200;
 		order: 99; // high number to make it always appear after the audio player
 		padding: 0 $player-grid-spacing;
 		margin-bottom: $player-grid-spacing;
-		color: $dark-gray-500;
 		font-size: $description-font-size;
 		line-height: 1.6;
+		color: $dark-gray-500;
+	}
+
+	.jetpack-podcast-player__container.has-secondary { // custom color.
+		.jetpack-podcast-player__track-description {
+			color: currentColor;
+		}
```

Comment:
I think you can remove the `.jetpack-podcast-player__container` part as well. The specificity is already high enough and `jetpack-podcast-player__container` doesn't render on the React version yet.